DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/TestRoutes/AccountTestRoutes.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using Xunit;
+using MvcRouteTester;
+using GitHubExtension.Security.WebApi.Library.Controllers;
+using System.Net.Http;
+using GitHubExtension.Models.CommunicationModels;
+using GitHubExtension.Constant;
+using System.Web.Http;
+using System.Text.RegularExpressions;
+
+namespace GitHubExtension.Security.Tests.TestRoutes
+{
+    public class AccountTestRoutes : TestRoutesConfig
+    {
+        [Fact]
+        public void AccountGetUserTest()
+        {
+            url = "/" + RouteConstant.apiAccount +
```

Comment:
So I suggest than move symbol "/" or to base class or extension method for asserting routes...